views/audit/README.md

```diff
@@ -0,0 +1,48 @@
+There are two views in this folder: and (query_audit_v2.sql in this folder)
```

Comment:
Can we add a few sample usage queries to make the README a bit more comprehensive?